app/src/main/java/org/fossasia/openevent/dbutils/DatabaseOperations.java

```diff
@@ -61,7 +63,7 @@
                 );
                 sessions.add(s);
             } catch (ParseException e) {
-                e.printStackTrace();
+                Timber.e("Parsing Error Occurred");
```

Comment:
You should also add the cause of parsing error. or maybe just the identifier of the method where it occured.